client/app/home/menu/menu.html

```diff
@@ -0,0 +1,9 @@
+<button ng-click="create()" ng-show="isState('home.menu')">Create new board</button>
+<div ui-view></div>
+
+<div class="container">
+  <div ui-sref="home.board({id: board.id})" class="board" ng-repeat="board in boards | orderBy:'-createdAt'">
```

Comment:
What the `ui-sref` directive does. On click:
1. Change state to `board`
2. Pass in parameter `id` equal to `board.id` to `board` state controller
3. The url is updated to `#/board/123` if `id = 123`